Makefile

```diff
@@ -306,16 +308,21 @@ endif
 
 ci.e2e.build.all:
 	WHAT="make build" make devkit.run
-	WHAT="./bin/konvoy-image build images/ami/centos-7.yaml -v 6" make devkit.run
+	WHAT="./bin/konvoy-image build images/ami/centos-7.yaml -v ${VERBOSITY}" make devkit.run
+	make docker.clean-latest-ami
+	WHAT="./bin/konvoy-image build images/ami/centos-8.yaml -v ${VERBOSITY}" make devkit.run
 	make docker.clean-latest-ami
-	WHAT="./bin/konvoy-image build images/ami/centos-8.yaml -v 6" make devkit.run
+	WHAT="./bin/konvoy-image build images/ami/sles-15.yaml --overrides overrides/nvidia.yaml -v ${VERBOSITY}" make devkit.run
 	make docker.clean-latest-ami
-	WHAT="./bin/konvoy-image build images/ami/centos-7.yaml --overrides overrides/nvidia.yaml -v 6" make devkit.run
+	WHAT="./bin/konvoy-image build images/ami/centos-7.yaml --overrides overrides/nvidia.yaml -v ${VERBOSITY}" make devkit.run
 	make docker.clean-latest-ami
-	WHAT="./bin/konvoy-image build images/ami/centos-8.yaml --overrides overrides/nvidia.yaml -v 6" make devkit.run
+	WHAT="./bin/konvoy-image build images/ami/centos-8.yaml --overrides overrides/nvidia.yaml -v ${VERBOSITY}" make devkit.run
+	make docker.clean-latest-ami
+	WHAT="./bin/konvoy-image build images/ami/sles-15.yaml --overrides overrides/nvidia.yaml -v ${VERBOSITY}" make devkit.run
```

Comment:
The first SLES build should be without the override for GPU support.

The Makefile also includes targets for `centos7` and `centos7-nvidia`. Can you please add similar for Suse?

Reply:
done ty